videocore6/driver.py

```diff
@@ -194,4 +206,4 @@ def execute(self, code, uniforms=None, timeout_sec=10):
         # XXX: Separate function
         for bo_handle in self.bo_handles:
             self.drm.v3d_wait_bo(bo_handle,
-                                 timeout_ns=int(timeout_sec / 1e-9))
+                                 timeout_ns=int(10 * timeout_sec / 1e-9))
```

Comment:
What is the purpose of this? Regarding the result from `test_clock` in `tests/test_qpu.py`, I think this parameter is not needed to be changed.

Reply:
Right. After my performance experiment, I forgot to restore it.